plugins/devices/frontend-java/src/main/java/com/freedomotic/jfrontend/JavaDesktopFrontend.java

```diff
@@ -175,7 +175,7 @@ protected void onCommand(final Command c)
             drawer.createCallout(callout1);
         } else if (c.getProperty("command") != null && c.getProperty("command").equals("GRAPH-DATA")) { // show GUI
             // get related object from address
-            EnvObjectLogic obj = (EnvObjectLogic) getApi().things().findByAddress("harvester", c.getProperty("event.object.address")).toArray()[0];
+            EnvObjectLogic obj = (EnvObjectLogic) getApi().things().findByAddress("harvester", c.getProperty("event.object.address"));
```

Comment:
Is an EnvObjectLogic guaranteed to be found? In any case, I think it'll be a good idea to check for null and log if so. Thoughts?